src/ert/scheduler/openpbs_driver.py

```diff
@@ -226,25 +263,64 @@ async def kill(self, iens: int) -> None:
 
     async def poll(self) -> None:
         while True:
-            if not self._jobs.keys():
-                await asyncio.sleep(_POLL_PERIOD)
-                continue
-
-            proc = await asyncio.create_subprocess_exec(
-                "qstat",
-                "-fxFjson",
-                *self._jobs.keys(),
-                stdout=asyncio.subprocess.PIPE,
-            )
-            stdout, _ = await proc.communicate()
-            if proc.returncode != 0:
+            if not self._jobs:
                 await asyncio.sleep(_POLL_PERIOD)
                 continue
 
-            stat = _Stat.model_validate_json(stdout)
-
-            for job_id, job in stat.jobs.items():
-                await self._process_job_update(job_id, job)
+            if self._non_finished_job_ids:
+                process = await asyncio.create_subprocess_exec(
+                    "qstat",
+                    "-x",
+                    *self._non_finished_job_ids,
+                    stdout=asyncio.subprocess.PIPE,
+                    stderr=asyncio.subprocess.PIPE,
+                )
+                stdout, stderr = await process.communicate()
+                if process.returncode not in {0, QSTAT_UNKNOWN_JOB_ID}:
+                    # Any unknown job ids will yield QSTAT_UNKNOWN_JOB_ID, but
+                    # results for other job ids on stdout can be assumed valid.
+                    await asyncio.sleep(_POLL_PERIOD)
+                    continue
+                if process.returncode == QSTAT_UNKNOWN_JOB_ID:
+                    logger.debug(
```

Comment:
Shouldn't this be a `logger.error`? Ert shouldn't ask qstat for jobs that it doesn't know about.

Reply:
I think I chose debug because of the volume of logging that this potentially could give (worst case is every realization every second second)

Reply:
And I am not convinced upfront that it always would be Ert that is at fault when/if this happens.